modules/cactus-form/src/SubmitButton.ts

```diff
@@ -0,0 +1,24 @@
+import { Button } from '@repay/cactus-web'
+import { formSubscriptionItems } from 'final-form'
+import React from 'react'
+
+import FormSpy from './FormSpy'
+
+export default FormSpy.withDefaults({
+  component: (props) => {
+    const propsCopy = { ...props }
+    delete propsCopy.form
+    formSubscriptionItems.forEach((sub) => {
+      delete propsCopy[sub]
+    })
+    return React.createElement(Button, propsCopy)
+  },
```

Comment:
If you remove the "always" Object.assign as I suggest in my other comment, I think you can simplify this to just
```suggestion
  component: Button,
```